server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,116 @@
+
+.Add Client
+image:{project_images}/add-client-oidc.png[Add Client]
+
+Enter in the `Client ID` of the client.  This should be a simple
+alpha-numeric string that will be used in requests and in the {project_name} database to identify the client.
+Next select `openid-connect` in the `Client Protocol` drop down box.
+Finally enter in the base URL of your
+application in the `Root URL` field and click `Save`.  This will create the client and bring you to the client `Settings`
+tab.
+
+.Client Settings
+image:{project_images}/client-settings-oidc.png[Client Settings]
+
+Let's walk through each configuration item on this page.
+
+*Client ID*
+
+The alpha-numeric string that will be used as the client identifier for OIDC requests.
+
+*Name*
+
+The display name for the client whenever it is displayed in a {project_name} UI screen.  You can localize
+the value of this field by setting up a replacement string value i.e. $\{myapp}.  See the link:{developerguide_link}[{developerguide_name}]
+for more information.
+
+*Description*
+
+The description of the client.  This can also be localized.
+
+*Enabled*
+
+When turned off, the client will not be allowed to request authentication.
+
+*Consent Required*
+
+If this is on, then users will get a consent page which asks the user if they grant access to that application.  It will also display the metadata that the client is interested in so that the user knows exactly what information the client is getting access to.
+If you have ever done a social login to Google, you'll often see a similar page.  {project_name} provides the same functionality.
+
+[[_access-type]]
+*Access Type*
+
+The type of OIDC client.
+
+_confidential_::
+  Confidential access type is for server-side clients that need to perform a browser login and require a client secret when they turn an access code into an access token,
+  (see https://tools.ietf.org/html/rfc6749#section-4.1.3[Access Token Request] in the OAuth 2.0 spec for more details). This type should be used for server-side applications.
+
+_public_::
+  Public access type is for client-side clients that need to perform a browser login. With a client-side application there is no way to keep a secret safe. Instead it is very important to restrict  access by configuring correct redirect URIs for the client.
+
+_bearer-only_::
+  Bearer-only access type means that the application only allows bearer token requests.
+  If this is turned on, this application cannot participate in browser logins.
+
+*Standard Flow Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Authorization Code Flow>>.
+
+*Implicit Flow Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Implicit Flow>>.
+
+*Direct Access Grants Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Direct Access Grants>>.
+
+*Root URL*
+
+If {project_name} uses any configured relative URLs, this value is prepended to them.
```

Comment:
This value adds a prefix to any configured URLS that  {project name} uses.

Do not use prepend. See Pg 354.